src/test/java/com/checkmarx/flow/cucumber/integration/pullrequest/updatecomments/UpdatePullRequestCommentsSteps.java

```diff
@@ -85,7 +96,13 @@ public UpdatePullRequestCommentsSteps(GitHubService gitHubService, GitHubPropert
         this.flowProperties = flowProperties;
         this.cxProperties = cxProperties;
         this.scaProperties = scaProperties;
+        this.authClient = authClient;
+        sastFile = toFullResourcePath(filePath);
+    }
 
+    private static File toFullResourcePath(String relativePath) throws IOException {
```

Comment:
we should create a utility for this, and unify the way we load files across all the tests.

Reply:
The existing TestUtils class looks very similar.

Reply:
changed to use the right method in TestUtils